pages/common/{.md

```diff
@@ -0,0 +1,36 @@
+# Curly brace
+
+> Multipurpose shell syntax.
+> More information: <https://www.gnu.org/software/bash/manual/bash.html#index-hash>.
+
+- Isolate variable names:
+
+`echo ${HOME}work`
+
+- Brace expand sequences:
+
+`echo {1..3} {a..c}{dir1,dir2,dir3}`
+
+- Check if `variable` is set before returning text:
+
+`echo ${variable:+variable is set and contains "$variable"`
```

Comment:
There needs to be a closing bracket here somewhere, right?

Reply:
Correct.